README.md

```diff
@@ -2,7 +2,10 @@
 
 ## Installation
 
-Copy the extension to `phpBB/ext/phpbb/skeleton`
+To install the source code from this repository, clone this extension repository to `phpBB/ext/phpbb/skeleton`:
```

Comment:
Well, for now, I'm just modifying the original text here. But once we have a release, this Readme will be re-written to point users towards releases. Only developers like us, who know what we're doing and want to contribute to this project should be cloning this repo.